packages/helpers/packages/ProxyHelpers/test/index.spec.js

```diff
@@ -0,0 +1,20 @@
+import * as ProxyHelpers from '../index.js';
```

Comment:
`.js` suffix reduce our codeFactor .

Reply:
since the name of the module is ProxyHelpers, it's better to call it by the full name.